moon-store-mac-os/moon-store-mac-os/Views/Main/MainViewViewModel.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  MainViewViewModel.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 12/15/24.
+//
+
+import Foundation
+
+final class MainViewViewModel: ObservableObject {
+    private let authenticationRepository: AuthenticationRepository = .init()
+    
+    func logout() {
+        do {
+            try authenticationRepository.logout()
+        } catch {
+            guard let error = error as? MSError else { return }
+            AlertPresenter().showAlert(type: .error, alertMessage: error.friendlyMessage)
```

Comment:
I think `AlertPresenter` should have all methods declared as static

Reply:
El alert presenter no deberia de tener esa instancia, como es una clase que al parecer no guarda estado es mejor ser un static